### DISCUSSION TITLE
bar chart possible?

### DISCUSSION THREAD

**Discussion by nqnga** [highfestiva/finplot#329] [Q&A]
Can I ask how can I do bar chart like this:
![image](https://user-images.githubusercontent.com/9681519/167268282-00c7d669-af4c-48ac-bb5e-642411032098.png)
Thanks!

**Comment by highfestiva** [comment]
Unfortunately not. It wouldn't be too hard to implement, but just haven't seen the point (yet, at least). If you'd like to take a shot at it, I'd be happy to include your pull request if it's clean and simple.
